### PR TITLE
Optimal use with modern Nim

### PR DESCRIPTION
**Pull request by momeemt**
Thank you for developing the freetype wrapper.
I found a few things that need to be fixed in order to use this wrapper comfortably with the current Nim.

1. The syntax of Nimble has changed.
1. The structure that the Nimble Package should take has changed.
1. The csize type has been deprecated.

These fixes make disruptive changes, so please check carefully!

### CODE REVIEW

File: freetype.nimble [hunk 1]
```diff
@@ -1,9 +1,10 @@
-[Package]
-name          = "freetype"
```
Comment (jangko) [inline freetype.nimble:2]: You can use `packageName` to replace `name`.

File: freetype.nimble [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
-version       = "0.1.1"
+# Package
+version       = "0.2.0"
 author        = "Andri Lim"
 description   = "FreeType2 library wrapper"
 license       = "MIT"
+srcDir        = "src"
```
Comment (jangko) [inline freetype.nimble:6]: please keep the directory structure as is. I don't like `src/freetype`, a simple `freetype` is ok.

File: src/freetype.nim [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import freetype/[freetype, ftadvanc, ftautoh, ftbbox, ftbdf, ftbitmap, ftbzip2, ftcache, ftcffdrv, ftcid, fterrdef, ftfntfmt, ftgasp, ftglyph, ftgxval, ftgzip, ftimage, ftimport, ftincrem, ftlcdfil, ftlist, ftlzw, ftmac, ftmm, ftmodapi, ftmoderr, ftotval, ftoutln, ftpfr, ftsizes, ftsnames, ftstroke, ftsynth, ftsystem, fttrigon, ftttdrv, fttypes, ftwinfnt, t1tables, ttnameid, tttables, tttags, ttunpat]
```
Comment (jangko) [inline src/freetype.nim:1]: this file should be put in root directory.

File: src/freetype/ftbbox.nim [hunk 4]
```diff
@@ -5,7 +5,7 @@
 #
 #-----------------------------------------
 
-import freetype, fttypes, ftimage
```
Comment (jangko) [inline src/freetype/ftbbox.nim:8]: good catch. nice.

**Review by jangko — CHANGES_REQUESTED** [review]

**Comment by neroist** [comment]
@jangko don't you think you should close this? Its been a year.

**Comment by jangko** [comment]
closing in favor of #4

**Pull request closed without merging**